tests/simd/basic_u.ml

```diff
@@ -0,0 +1,272 @@
+open Stdlib
+
```

Comment:
You've duplicated the tests for boxed simd, which is great, but we're also going to need a bunch more tests: typing tests for the new type and sort, and possibly some more runtime tests.

You will notice for each sort we have a test directory `ocaml/testsuite/tests/typing-layouts-<sort>`.  Please look, for example, at the `bits64` tests and duplicate them for the new sort.  The runtime tests in `unboxed_bits64s.ml` are somewhat duplicative of the runtime tests you have here, and may not need to be duplicated (but check you've got all those cases here).  The typing tests should ~all be duplicated, and consider if any new typing tests should be added for the new types.  (E.g., add a test showing if you try to use `int64x2#` where `int32x4#` is expected, you get a type error).

Reply:
I've duplicated the typing tests under `typing-layouts-vec128`, but the runtime tests (`test_int64_u.ml`/`alloc.ml`/`unboxed_bits64.ml`) can't be directly ported, and I think these are covered by the unboxed SIMD tests under `tests/simd`.

Reply:
Also note the new tests include fatal errors when using vec128 fields; this will be fixed in a following PR